packages/sources/svelte/src/components/structure/VtmnCard/VtmnCard.svelte

```diff
@@ -59,7 +65,13 @@
   {/if}
   <div class={componentContentClass}>
     {#if title}
-      <span class="vtmn-card_content--title">{title}</span>
+      <span
+        role={headingLevel && 'heading'}
+        aria-level={headingLevel && headingLevel}
+        class="vtmn-card_content--title"
+      >
+        {title}
+      </span>
```

Comment:
A possible semantic alternative: 
```
const Heading = `h${headingLevel}`;
```
and then
```
<Heading class="vtmn-card_content--title">
       {title}
</Heading>
```

Reply:
Good idea

Reply:
Not possible, to do that it's my first commit with the `<svelte:element>`